lib/constants.rb

```diff
@@ -1090,6 +1090,10 @@ module Constants
    }
    BROWSABLE_CALLNUMBERS = ["LC", "DEWEY", "ALPHANUM"]
    LIBRARY_INSTRUCTIONS = {
+     'ART' => {
+       heading: 'Library closed June25-August23',
```

Comment:
Original text is what PB provided, but yeah, it's better w spaces. Let's do that.